src/views/AnalysisModule/Analysis/index.tsx

```diff
@@ -60,15 +192,79 @@ function Analysis(props: ComponentProps) {
                     </QuickActionButton>
                     <QuickActionButton
                         className={styles.button}
+                        onClick={handleDeleteAnalysis}
                     >
                         <Icon name="delete" />
                     </QuickActionButton>
                 </div>
             )}
+            contentClassName={styles.pillarContent}
         >
-            Content
+            <div className={styles.content}>
+                <div className={styles.contentItem}>
+                    <TextOutput
+                        className={styles.textOutput}
+                        labelClassName={styles.label}
+                        valueClassName={styles.value}
+                        label={_ts('analysis', 'teamLead')}
+                        value={teamLeadName}
+                        noColon
+                    />
+                </div>
+
+                <div className={styles.contentItem}>
+                    <h3 className={styles.subHeading}>
+                        {_ts('analysis', 'pillarAssignments')}
+                    </h3>
+                    <ListView
+                        data={analysisPillar}
+                        renderer={PillarListItem}
+                        rendererParams={pillarListRendererParams}
+                        keySelector={keySelector}
+                    />
+                </div>
+            </div>
+            <div
+                className={styles.pillarAnalyses}
+            >
+                <Button
+                    className={styles.accordionButton}
+                    icons={(
+                        <Icon name={isExpanded
+                            ? 'chevronUp'
+                            : 'chevronDown'}
+                        />
+                    )}
+                    onClick={handleClick}
+                >
+                    {_ts('analysis', 'pillarAnalysisCount', { count: pillarResponse?.count })}
+                </Button>
+                {isExpanded && (
+                    <>
+                        <div className={styles.pillarAnalysisContent}>
+                            <ListView
+                                className={styles.pillarList}
+                                data={analysisPillar}
+                                keySelector={keySelector}
+                                renderer={AnalysisPillar}
+                                rendererParams={analysisPillarRendererParams}
+                                pending={pillarPending}
+                            />
+                        </div>
+                        {pillarResponse && pillarResponse.count > MAX_ITEMS_PER_PAGE && (
```

Comment:
This logic can also be moved inside the pager component. (If we are using the new one)

Reply:
Lets skip this for now